composer.json

```diff
@@ -39,9 +39,13 @@
         "myclabs/deep-copy": "~1.3",
         "ext-dom": "*",
         "ext-json": "*",
-        "ext-pcre": "*",
-        "ext-reflection": "*",
```

Comment:
As noted by @sebastianbergmann in #2288 these extensions always present in PHP and he does not see any reason to check for them in composer.json. I agree with him,